doc/manuals/user/context_providers.md

```diff
@@ -53,79 +53,51 @@ curl localhost:1026/v2/registrations -s -S -H 'Content-Type: application/json' -
   },
   "provider": {
     "http": {
-      "url": "http://sensor48.mycity.com/v1"
-    },
-    "legacyForwarding": true
+      "url": "http://sensor48.mycity.com/v2"
+    }
   }
 }
 EOF
 ```
       
       
-* Next, consider that a client queries the temperature  (message number 2).
+* Next, consider that a client queries the temperature (message number 2).
 
       
 ```
 curl localhost:1026/v2/entities/Street4/attrs/temperature?type=Street -s -S \
     -H 'Accept: application/json' -d @- | python -mjson.tool
 ``` 
 
-* Orion doesn't know the Street 4 temperature, but it knows (due to
+* Orion doesn't know the Street4 temperature, but it knows (due to
   the registration in the previous step) that the Context Provider at
-  <http://sensor48.mycity.com/v1> does know about the Street 4 temperature, so it forwards the query
+  <http://sensor48.mycity.com/v2> does know about the Street4 temperature, so it forwards the query
   (message number 3) to the URL
-  <http://sensor48.mycity.com/v1/queryContext> (i.e. the URL used in
-  the Providing Application field at registration time, plus the
-  "/queryContext" operation). Note that the query is forwarded using
-  NGSIv1 format although the original request from the client used NGSIv2
-  (NGSIv1 is deprecated, but we need to do so as there is yet lack of support for NGSIv2 based forwarding, see
-  [this issue about it](https://github.com/telefonicaid/fiware-orion/issues/3068)).
+  <http://sensor48.mycity.com/v2/entities> (i.e., the URL used in
+  the Providing Application field at registration time, and adding "/entities" to the URL PATH).
```

Comment:
...the URL used in the `url` field

(Providing Application is the old one in NGSIv1)

Reply:
Fixed in aef7cb8